sdk/eventhub/azure-eventhub/azure/eventhub/_consumer.py

```diff
@@ -83,7 +83,7 @@ def __init__(
         event_position = kwargs.get("event_position", None)
         prefetch = kwargs.get("prefetch", 300)
         owner_level = kwargs.get("owner_level", None)
-        keep_alive = kwargs.get("keep_alive", None)
+        keep_alive = kwargs.get("keep_alive", 30)
```

Comment:
for symmetry with the producer, would we want to add this at the consumer client level and pass it in when creating consumers or is this preferred?

Reply:
this isnt exposed on the consumer client docstrings so I put it in the consumer

Reply:
if we want to align the two should we add keep alive to the docstring

Reply:
this is fine w/ me, but just checking